engine/spec.go

```diff
@@ -114,6 +115,14 @@ type (
 		FieldPath string `json:"field_path,omitempty"`
 	}
 
+	// VolumeClaim ...
+	VolumeClaim struct {
+		ID        string `json:"id,omitempty"`
+		Name      string `json:"name,omitempty"`
+		ClaimName string `json:"claimName,omitempty"`
+		ReadOnly  bool   `json:"readOnly,omitempty"`
```

Comment:
snake case please :)

Reply:
ups, I forgot to update this

Reply:
I also forgo to conver to snake case the readOnly label, now it's done.